File: d_rank/d_rank_level_up_problems/php/loop/boss.php
<?php
    for ($i = 1; $i <= 100; $i++) {
        if ($i % 15 == 0) {
            echo 'FizzBuzz'.PHP_EOL;
        }
        else if ($i % 3 == 0) {
            echo 'Fizz'.PHP_EOL;
        }
        else if ($i % 5 == 0) {
            echo 'Buzz'.PHP_EOL;
        }
        else {
            echo $i.PHP_EOL;
        }
    }
?>

File: d_rank/d_rank_level_up_problems/c/loop/boss.c
// Paiza 問題集 Dランクレベルアップメニュー FizzBuzz C編
// https://paiza.jp/works/mondai/d_rank_level_up_problems/d_rank_level_up_problems__loop_boss
#include <stdio.h>

#define FIZZ(n) ((n) % 3 == 0)
#define BUZZ(n) ((n) % 5 == 0)

int main()
{
    for (int i = 1; i <= 100; i++) {
        if (FIZZ(i) && BUZZ(i)) {
            printf("FizzBuzz\n");
        } else if (FIZZ(i)) {
            printf("Fizz\n");
        } else if (BUZZ(i)) {
            printf("Buzz\n");
        } else {
            printf("%d\n", i);
        }
    }

    return 0;
}